tests/e2e/page-layouts-modal.spec.js

```diff
@@ -0,0 +1,13 @@
+const {test, expect} = require('@playwright/test');
+import {login} from './tools/lib/login';
+
+test('checks if the welcome modal on the editor is present and closed when the button is clicked', async ({page, context}) => {
+  test.setTimeout(240 * 1000);
+  await page.goto('./');
+  await login(page, context);
+  await page.goto('./wp-admin/post-new.php?post_type=page');
+  const locator = page.locator('.components-modal__screen-overlay');
+  await expect(locator).toBeVisible();
+  await page.getByRole('button', {name: 'Close dialog'}).click();
```

Comment:
I tested locally and it failed here. Thus, I changed to `await page.getByRole('button', {name: 'Close'}).click();` and it finally passed.

![Screenshot 2023-05-03 at 08 51 39](https://user-images.githubusercontent.com/77975803/235908293-9fcc87eb-9a62-4397-8b9c-2e0a9c1a310c.png)


What do you think?